FraihmworkRulesEngineApi.yaml

```diff
@@ -11,7 +11,7 @@ externalDocs:
   url: http://swagger.io
 
 servers:
-- url: https://<site>.fraihmwork.resilienx.com/api
```

Comment:
I'm curious about this change - I think it's a good call because tools that let you interact with the spec like SwaggerUI and others can't do anything with the invalid URL, but this is different from all of our other specs in this repository.